astropy/modeling/powerlaws.py

```diff
@@ -305,7 +305,7 @@ def evaluate(x, amplitude, x_break, alpha_1, alpha_2, delta):
             t = np.exp(logt[i])
             r = (1. + t) / 2.
             f[i] = amplitude * xx[i] ** (-alpha_1) \
-                   * r ** ((alpha_1 - alpha_2) * delta)
+                * r ** ((alpha_1 - alpha_2) * delta)
```

Comment:
If it's just one line is line length < 100 ?

Reply:
It is weird that it was line wrapped at all. I just corrected the indentation error without really considering what was going on.

Reply:
See 84b3922